### PR TITLE
Rectify sequence of Array challenges

### PR DESCRIPTION
**Pull request by caje731**
On completion of "Accessing Array Values" the next challenge mentioned is "Looping Through Arrays".
But the sequence of challenges is such that the latter comes before the former.
This commit fixes it.

### CODE REVIEW

**Comment by steakunderscore** [comment]
I also had this thought. Although, what would be more preferable to a programming newbie? Some time indexing by zero can be a bit confusing. Which then it might make more sense to have the looping challenge first. Thoughts?

**Comment by caje731** [comment]
I see your point. But personally, I believe learning loops after learning indexing is more natural.
Once I have learnt how to access an element using its index, understanding loops become easy since the loop counter readily lends itself as the array index. As a programming newbie, I'll be delighted by that discovery and it's use.
What do you think ?

**Comment by caje731** [comment]
I see your point. But personally, I believe learning loops after learning indexing is more natural.
Once I have learnt how to access an element using its index, understanding loops become easy since the loop counter readily lends itself as the array index. As a programming newbie, I'll be delighted by that discovery and it's use.
What do you think ?

**Comment by SomeoneWeird** [comment]
Maybe we can add a challenge that tries to explain how arrays are indexed before we get into the other array challenges?

**Comment by caje731** [comment]
@SomeoneWeird I believe that's the purpose of the already existing challenge 'Accessing Array Values', isn't it ?

**Comment by SomeoneWeird** [comment]
You are certainly correct. Maybe we need to reword it to try and make it clearer?

**Comment by caje731** [comment]
@SomeoneWeird  mmm ...  I'm not sure whether a re-wording would be necessary, because I felt "Accessing Array Values" conveyed the idea clearly. I would rather only have the challenge moved up front before all other array-access challenges.
Do you agree ?

**Comment by SomeoneWeird** [comment]
Ah, I see what you're saying - my bad. I agree.

**Comment by caje731** [comment]
Great, cheers!
